### ISSUE TITLE
Day Planner will auto remove any words before `-- {any chars}` each line.

### ISSUE THREAD

**Issue by edentsai** [lynchjames/obsidian-day-planner#44]
Hi, I found a bug that Day Planner will auto remove all words before `-- {any chars}`
it trigged by any line ending with double hypens and any chars.

for examples:

```diff
  # Before (What I write)
  nothing happen here that ending with double hyphens only --
- this will be changed by Day Planner that ending with double hyphens and any chars -- bug

  # After (caused by the bug)
  nothing happen here that ending with double hyphens only --
+ - bug
```
```diff
- -- another case
+ - another case
```

```diff
- hello world -- 12345
+ - 12345
```

I recorded a GIF for show the bug:

![day-planner-will-remove-any-words-before-double-hyphens-and-any-chars](https://user-images.githubusercontent.com/5071526/101790079-00f48200-3b3d-11eb-8703-f52a9578aa41.gif)

**Comment by edentsai** [comment]
oh, this is same issue with https://github.com/lynchjames/obsidian-day-planner/issues/43

**Comment by lynchjames** [comment]
This should now be fixed in [0.5.7](https://github.com/lynchjames/obsidian-day-planner/releases/tag/0.5.7).
